docs/interlock_data.md

```diff
@@ -9,6 +9,7 @@ compatibility.
 |Label|Extensions Supported|
 |----|----|
 |`interlock.ext.name`               | internal |
+|`interlock.ext.service.name`       | internal |
```

Comment:
'nit: i think we should keep this to three levels (like `interlock.ext.service_name` perhaps.